src/module/comment.js

```diff
@@ -0,0 +1,55 @@
+// https://api.tvmaze.com/seasons/1/episodes
+import close from '../images/close.png';
+import getComment from './getComment.js';
+import counter from './commentcounter.js';
+import createFormContainer from './commentform.js';
+import getData from './getData.js';
+
+const main = document.querySelector('.content');
+const popup = document.querySelector('.popup-dialog');
+const formContainer = document.createElement('form');
+const commentContainer = document.createElement('div');
+commentContainer.className = 'comment-container';
+
+formContainer.className = 'form';
+
+const showPopupDialog = async (id) => {
+  const commentCounter = document.createElement('h2');
+  const result = await getData();
+  // eslint-disable-next-line no-unused-vars
+  const index = +id - 1;
+
+  const newMovies = result[5];
+  const comments = await getComment(newMovies.id);
+  const totalComment = await counter(newMovies.id);
+  main.style.display = 'none';
+  popup.style.display = 'block';
+  popup.innerHTML = `
+        <div class="popup-top">
+            <img src="${newMovies.image.medium}" class='popup-image' alt="image">
+             <img src="${close}" class='close-btn' alt="image">
+        </div>
+            <h2 class="title"> ${newMovies.name}</h2>
+        <ul class="detail">
```

Comment:
- [ ] Kindly, make sure that you add the details about the movie in your popup window. you should add also the summary about the season episode. 
```html
<h2 class="title"> ${newMovies.name}</h2>
 <p class="summary"> ${newMovies.summary}</p>
```

Reply:
Thanks for the review.
We just added the details about the movie in the popup window.